src/cmd/apply-as-apps.ts

```diff
@@ -43,6 +43,19 @@ const getAppName = (release: HelmRelease): string => {
 }
 
 const getArgocdAppManifest = (release: HelmRelease, values: Record<string, any>, otomiVersion) => {
+  const releases = [
+    'tempo',
+    'thanos',
+    'cert-manager',
+    'cloudnative-pg',
+    'prometheus-operator',
+    'tekton-pipelines',
+    'tekton-triggers',
+    'jaeger-operator',
+    'otel-operator',
```

Comment:
Would this fix also work for `ingress-ingress-nginx-platform`?

Reply:
That is covered by this part: 
`const releaseNameMatches = releases.includes(release.name) || /^nginx-.*/.test(release.name)`
It also adds the annotation for all nginx-*